docker/bench-all.sh

```diff
@@ -13,8 +13,15 @@ source /opt/conda/etc/profile.d/conda.sh
 conda activate ucx
 
 cd ucx-py/
+
 # Benchmark using command-line provided transports or else default
-for tls in ${@:-"tcp" "all"}; do
+if [[ ${#@} -eq 0 ]]; then
+    transport_types=(tcp all)
+else
+    transport_types=("${@}")
+fi
+
+for tls in "${transport_types[@]}"; do
```

Comment:
Here's what `shellcheck` was unhappy about:

```text
In docker/bench-all.sh line 17:
for tls in ${@:-"tcp" "all"}; do
           ^---------------^ SC2068 (error): Double quote array expansions to avoid re-splitting elements.
```

Now I think it's highly unlikely that you're intentionally pass any value with spaces here, so if you'd prefer I'd be happy to revert this change and just stick a `# shellcheck disable` comment on it.

Reply:
This seems like a reasonable change to me